geolocator_platform_interface/lib/src/enums/location_accuracy.dart

```diff
@@ -20,5 +20,10 @@ enum LocationAccuracy {
 
   /// Location accuracy is optimized for navigation on iOS and matches the
   /// [LocationAccuracy.best] on Android
-  bestForNavigation
+  bestForNavigation,
+
+  /// Location accuracy is reduced for iOS 14+ devices, matches the
+  /// [LocationAccuracy.low] and [LocationAccuracy.lowest] on Android and
+  /// matches [LocationAccuracy.lowest] on devices using iOS 13 or below.
```

Comment:
nit: we also support other platforms then iOS and Android so we should reflect this in our documentation. Maybe change this to:

```suggestion
  /// Location accuracy is reduced for iOS 14+ devices, matches the
  /// [LocationAccuracy.lowest] on iOS 13 and below and all other platforms.
```